README.md

```diff
@@ -345,7 +345,7 @@ To define a test, use `defineTest` from the `testUtils` module:
 
 ```js
 jest.autoMockOff();
-const defineTest = require('jscodeshift/src/testUtils').defineTest;
+const defineTest = require('jscodeshift/dist/testUtils').defineTest;
```

Comment:
Since src and dist are the same files (assuming `prepublish` was recently run), what are the advantages of recommending dist over src here?